src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5351,11 +5368,14 @@ public void StartEvent(string eventName, EventAttribute eventAttribute)
             numParams = 0;
             byteArrArgIndices = null;
 
-            events.Append("  <event value=\"").Append(eventAttribute.EventId).
-                 Append("\" version=\"").Append(eventAttribute.Version).
-                 Append("\" level=\"");
-            AppendLevelName(events, eventAttribute.Level);
-            events.Append("\" symbol=\"").Append(eventName).Append('"');
+            if (events != null)
+            {
+                events.Append("  <event value=\"").Append(eventAttribute.EventId).
```

Comment:
Here you check if events is null byut then other places like line 5387 you use the `events?.` syntax which accomplishes the same thing. Both are correct, but we should use the same syntax across the whole change for consistency.